src/lsst/cmservice/web_app/app.py

```diff
@@ -250,16 +250,20 @@ async def get_script(
     session: async_scoped_session = Depends(db_session_dependency),
 ) -> HTMLResponse:
     try:
-        script_details = await get_script_by_id(session, script_id)
+        script_details = await get_script_by_id(
+            session,
+            script_id=script_id,
+            campaign_id=campaign_id,
+            step_id=step_id,
+            group_id=group_id,
+            job_id=job_id,
+        )
+        print(script_details)
```

Comment:
Do you want to remove the print statement?   I'm fine either way...

Reply:
Thanks for the reminder. I thought I have removed all print statements already